src/index.html

```diff
@@ -2,11 +2,77 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta 
+      name="viewport" 
+      content="width=device-width,
+      initial-scale=1.0"
+    >
```

Comment:
```suggestion
      content="
        width=device-width,
        initial-scale=1.0
      "
    >
```
Если у одного атрибута несколько значений, то тебе стоит каждое значение переносить на новую строку:)

Reply:
Оля,привет.Можешь напиать адресс своей єлектронной почты,или как с тобой можно было бы связаться?